tests/testsuite/namespace.go

```diff
@@ -87,7 +87,7 @@ func CleanNamespaces() {
 		}
 
 		// Clean namespace labels
-		err = libnet.RemoveAllLabelsFromNamespace(virtCli, namespace)
```

Comment:
Why can't we just modify `RemoveAllLabelsFromNamespace`?

Reply:
Two reasons:
- Cyclic import testsuite <-> libnet
- Wouldn't really stay true to it's name `RemoveAllLabelsFromNamespace` (we could argue it's not doing this today)

Reply:
Sure but if my local grep doesn't lie to me `RemoveAllLabelsFromNamespace` seems to be redundant now...

Reply:
Yep, it's definitely telling you the truth :D

Reply:
done